smart-contracts/assembly/contracts/FT/token-commons.ts

```diff
@@ -42,3 +42,32 @@ export function _setBalance(address: Address, balance: u256): void {
 function getBalanceKey(address: Address): StaticArray<u8> {
   return stringToBytes(BALANCE_KEY + address.toString());
 }
+
+/**
+ * Sets the allowance of the spender on the owner's account.
+ *
+ * @param owner - owner address
+ * @param spenderAddress - spender address
+ * @param amount - amount to set an allowance for
+ */
+export function _approve(
+  owner: Address,
+  spenderAddress: Address,
+  amount: u256,
+): void {
+  const key = stringToBytes(owner.toString().concat(spenderAddress.toString()));
```

Comment:
Do we risk some kind of storage collision here? Should we add a specific prefix such as "allow" ?

Reply:
Risk is limited, but I think this is a good practice, especially if we implement a method that allows iterating keys. For example, if we want to iterate all keys that start with some prefix.